install/installer.class.php

```diff
@@ -719,295 +719,450 @@ static public function getPolicies() {
+            'group'                                => 'phone',
+            'type'                                 => 'bool',
+            'type_data'                            => '',
+            'unicity'                              => 1,
+            'plugin_flyvemdm_policycategories_id'  => 8,
+            'comment'                              => __('Disable automatic answer to incoming phone calls', 'flyvemdm'),
+            'default_value'                        => '0',
+            'recommended_value'                    => '0',
+         ],
+
+         [
+            'name'                                 => __('Disable voice dicatation', 'flyvemdm'),
+            'symbol'                               => 'disableVoiceDictation',
+            'group'                                => 'connectivity',
+            'type'                                 => 'bool',
+            'type_data'                            => '',
+            'unicity'                              => 1,
+            'plugin_flyvemdm_policycategories_id'  => 5,
+            'comment'                              => __('Disable voice dicatation', 'flyvemdm'),
+            'default_value'                        => '0',
+            'recommended_value'                    => '0',
+         ],
 
+         [
+            'name'                                 => __('Disable NFC', 'flyvemdm'),
+            'symbol'                               => 'disableNfc',
+            'group'                                => 'connectivity',
+            'type'                                 => 'bool',
+            'type_data'                            => '',
+            'unicity'                              => 1,
+            'plugin_flyvemdm_policycategories_id'  => 5,
+            'comment'                              => __('Disable Near Field Contact', 'flyvemdm'),
+            'default_value'                        => '0',
+            'recommended_value'                    => '0',
+         ],
+
+         [
+            'name'                                 => __('Disable hotspot and tethering', 'flyvemdm'),
+            'symbol'                               => 'disableHostpotTethering',
+            'group'                                => 'connectivity',
+            'type'                                 => 'bool',
+            'type_data'                            => '',
+            'unicity'                              => 1,
+            'plugin_flyvemdm_policycategories_id'  => 5,
+            'comment'                              => __('Disable hotspot and tethering', 'flyvemdm'),
+            'default_value'                        => '0',
+            'recommended_value'                    => '0',
+         ],
+
+         [
+            'name'                                 => __('Disable new VPN profiles', 'flyvemdm'),
+            'symbol'                               => 'disableHostpotTethering',
```

Comment:
@btry Please check this item it duplicated

Reply:
Fixed here: https://github.com/flyve-mdm/flyve-mdm-glpi-plugin/pull/170